site/src/components/WorkspaceActions/WorkspaceActions.tsx

```diff
@@ -18,7 +17,7 @@ import { ButtonMapping, ButtonTypesEnum, WorkspaceStateActions } from "./constan
  * so check whether workspace job status has reached completion (whether successful or not).
  */
 const canAcceptJobs = (workspaceStatus: WorkspaceStatus) =>
-  ["started", "stopped", "deleted", "error", "canceled"].includes(workspaceStatus)
+  ["running", "stopped", "deleted", "failed", "canceled"].includes(workspaceStatus)
```

Comment:
Nice catch! This approach makes it easy to fall out of sync with valid `WorkspaceStatus` values. Here's one way we could fix this:

```typescript
const CAN_ACCEPT_JOBS_STATUS: Array<
  Extract<
    WorkspaceStatus,
    "running" | "stopped" | "deleted" | "failed" | "canceled"
  >
> = ["running", "stopped", "deleted", "failed", "canceled"];
const canAcceptJobs = (workspaceStatus: WorkspaceStatus) => CAN_ACCEPT_JOBS_STATUS.includes(workspaceStatus)
```

[TypeScript Playground](https://www.typescriptlang.org/play?#code/C4TwDgpgBA6g9gJwNYGcwEMDGEDKx3ACuKUAvAFBRQA+UARJugHbYA2EAJnZTfYyxFYBLJgHNuVWnQ6CIwThN7TZwEeJ5SAZuiHsuG+pCYc1iqQkJMmpg3RT4EqsWfr24YSPsmvg7sKYBucnJMOCZ7KABhAEEAOQB9aMjIgFEABQAVeIApAHkAIRx4nAzojIBVHAAuKGiEBHQQAB4eFIAPYAbMYBaqKnhkNCxcfCIUABoeKjoLK1MlNw8FJRl2eS4lbV1lqX42BR4APnJDsigAbRnLa2dxnz8FO+U1x-otvTonvdkuAF0g0LhYBQfjRTDYMDAbJwABGJFIUAAFAB3RCoDDYPAEYg1Abo4ZYsYASjIxyoMQSSVSmRyBSKJTKlQAdCJMKxCDIUCi0UNMaNiESAWEIqDwRBIWdRRCobCuXR3gpBSFhcD+Ew4MAwdLJcwteKZXDEQxmGxTErleE4Owmaw4KJEQADYAACyEJBQzrghFYHCgMOgnUIEBqABIAN5S-UAXwdSsBKCtEBtdsdLrdUA9Xp9fug2lYKGDUHDao1eshMaVQA)

Reply:
I'm actually fixing this in a future refactor, thanks for pointing it out!